README.md

```diff
@@ -19,46 +19,48 @@
       read_interval 2
       tag winevt.raw
       <storage>
-        @type local             # @type local is default.
-        persistent true         # default is true. If you want to use on-memory storage, set false.
+        @type local             # @type local is the default.
+        persistent true         # default is true. Set to false to use in-memory storage.
         path ./tmp/storage.json # This is required when persistent is true.
-                                # Or, please consider to use <system> section's root_dir parameter.
+                                # Or, please consider using <system> section's `root_dir` parameter.
       </storage>
     </source>
 
 #### parameters
 
 |name      | description |
 |:-----    |:-----       |
-|channels   | (option) 'applicaion' as default. one or combination of {application, system, setup, security}. If you want to read setup or security, administrator priv is required to launch fluentd.  |
-|read_interval   | (option) a read interval in second. 2 seconds as default.|
-|from_encoding  | (option) an input characters encoding. nil as default.|
-|encoding   | (option) an output characters encoding. nil as default.|
-|`<storage>`|Setting for storage plugin for recording read position like in_tail's pos_file|
+|`channels`   | (option) 'application' as default. One or more of {'application', 'system', 'setup', 'security'}. If you want to read 'setup' or 'security' logs, you must launch fluentd with administrator privileges.|
+|`keys`   | (option) A subset of [keys](#read-keys) to read. Defaults to all keys.|
+|`read_interval`   | (option) read interval in seconds. 2 seconds as default.|
+|`from_encoding`  | (option) input character encoding. `nil` as default.|
+|`encoding`   | (option) output character encoding. `nil` as default.|
+|`<storage>`| Setting for `storage` plugin for recording read position like `in_tail`'s `pos_file`.|
 
 #### read keys
-This plugin reads follows from Windws Event Log. No customization is allowed for now.
+This plugin reads the following fields from Windows Event Log entries. Use the `keys` configuration option to select a subset. No other customization is allowed for now.
 
 |key|
 |:-----    |
-|record_number   |
-|time_generated|
-|time_written   |
-|event_id   |
-|event_type   |
-|event_category   |
-|source_name   |
-|computer_name  |
-|user   |
-|description   |
+|`record_number` |
+|`time_generated`|
+|`time_written`  |
+|`event_id`      |
+|`event_type`    |
+|`event_category`|
+|`source_name`   |
+|`computer_name` |
+|`user`          |
+|`description`   |
+|`string_inserts`|
 
 ## Etc.
-'read_from_head' is not supporeted currently.You can read newer records after you start first.
+'`read_from_head`' is not supported currently. You can read newer records after you start first.
```

Comment:
I checked this parameter behavior with current master.
This option is supported in current master.
We can remove this.